docs/docs/cli.md

```diff
@@ -74,6 +74,24 @@ my-package
     └── test_my_package.py
 ```
 
+If you want to use `rst` instead of `md` as the `README` format, you can use the `--readme` option:
+```bash
+poetry new my-package --readme rst
+```
+
+That will create a folder structure as follows:
```

Comment:
I think printing the tree structure again is unnecessary. Maybe just add a short comment that a `README.rst` will be generated.